### DISCUSSION TITLE
SVG file path support

### DISCUSSION THREAD

**Discussion by ghost** [Chlumsky/msdf-atlas-gen#40] [Ideas]
Would appreciate support for SVG file paths. This is supported in msdfgen, so why not here too?

**Comment by Chlumsky** [comment]
Because this is a tool specifically made for fonts. I understand how this feature could be helpful, but unfortunately it is too complex of a use case for this project. Unlike font glyphs, which are guaranteed to be at the same scale and have unique indices, specifying the exact intent for a bunch of SVG files through command line interface would be complicated, not to mention potentially throwing in a font file as well. The output formats are also designed to represent font glyphs and not just random SVG files, so relaying the information of which SVG is where in the atlas would also be a problem.

As this is open source, feel free to modify your copy of the project to suit your needs. I believe there are also tools which generate a font from SVG files, which could be an alternative approach.
